README.rst

```diff
@@ -108,6 +108,20 @@ Logo
 
 scikit-bio's logo was created by `Alina Prassas <http://cargocollective.com/alinaprassas>`_.
 
+Citing scikit-bio
+-----------------
+
+If you use scikit-bio for any published research, please include the following
+citation:
+
+> @software{skbio2020,
```

Comment:
2020?

Can the DOI be included? See https://zenodo.org/record/8209901

Reply:
Should we just add the link to the DOI?